doc/simastrom.tex

```diff
@@ -30,7 +30,7 @@
 \def\rm{\normalfont \rmfamily}
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
-\title{Documentation of the meas/simastrom package}
+\title{Documentation of the jointcal package}
```

Comment:
Yes, this document is just about the WCS part of the algorithm. I'd rather not mess with the document, since from what I've heard it does not actually line up with the way the code is currently implemented (that's a separate issue).